com.virustotal.VirusTotalUploader.yml

```diff
@@ -0,0 +1,55 @@
+id: com.virustotal.VirusTotalUploader
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: '5.15'
+command: VirusTotalUploader
+finish-args:
+  - --device=dri
+  - --filesystem=host:ro
```

Comment:
wouldn't `home:ro` be enough?

Reply:
OK, but since there is not much difference from security side maybe instead of host/home even better to list all `xdg-*` dirs, removable devices `filesystem=/media:ro`, `filesystem=/mnt:ro`, `filesystem=/run/media` and probably `/opt`?

Reply:
@hfiguiere I think not, a common use case is to check files on removable drives, so `host` can be replaced with `home` + `/run/media` + `/media`, which is largely the same as `host`.

Reply:
Probably `filesystem=xdg-data:ro` and `filesystem=xdg-cache:ro` to much and should removed. Need your opinion here.

Reply:
Well, if the goal is secure-by-default, then it's certainly too much, as `~/.local/share` and `~/.cache` contain a lot of sensitive data.

Reply:
Really, the proper way is to use portals. Direct filesystem permissions are just hit&miss for non-obvious apps like this one. Why wouldn't user want to upload a file directly from their top-level home directory here?

Reply:
[This is apparently a QT 5 app](https://github.com/VirusTotal/qt-virustotal-uploader#generic-instructions-for-compiling-and-install), does that include the relevant portals?

Reply:
I have removed all filesystem permissions and it seems to work fine for me.

Reply:
@barthalion which permission settings you advice using here? So maybe allow just `Downloads` if portals works in this app?

Reply:
No flesystem permissions are needed when portal is used.

Reply:
@barthalion You can select file from the app menu even if no filesystem permissions, yes, but drag-and-dropping files to the app window won't work.

Reply:
Yep. Tested now and portals works in this app, but DnD not if we remove all FS permissions. That's why i mentioned in OP why we need more wide restriction since app even have title with DnD mention. `host` is to much for sure, so can we find middle ground here?

Reply:
I'd say `host:ro` is fine. Otherwise, we're guaranteed to have users for whom DnD would be broken, no matter how many smaller permissions we add.

Reply:
After some discussion reverted `host:ro` with some commentary. We ready for merge it?